test/evergreen_develop.yml

```diff
@@ -71,6 +53,37 @@ buildvariants:
       execution_tasks:
       - ".wt2853-perf"
 
+- <<: *perf-tasks-template
+  name: ubuntu2004-perf-tests
+  display_name: Ubuntu 20.04 Performance tests
+  run_on:
+    - ubuntu2004-medium
+  expansions: &ubuntu2004-perf-tests-expansions
+    test_env_vars:
+      WT_TOPDIR=$(git rev-parse --show-toplevel)
+      WT_BUILDDIR=$WT_TOPDIR/cmake_build
+      LD_LIBRARY_PATH=$WT_BUILDDIR:$WT_BUILDDIR/test/utility/
+    CC_OPTIMIZE_LEVEL: -DCC_OPTIMIZE_LEVEL=-O3
+    HAVE_DIAGNOSTIC: -DHAVE_DIAGNOSTIC=0
+    CMAKE_TOOLCHAIN_FILE: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_stable_gcc.cmake
+    CMAKE_INSTALL_PREFIX: -DCMAKE_INSTALL_PREFIX=$(pwd)/cmake_build/LOCAL_INSTALL
+    python_binary: '/opt/mongodbtoolchain/v4/bin/python3'
+    pip3_binary: '/opt/mongodbtoolchain/v4/bin/pip3'
+    virtualenv_binary: '/opt/mongodbtoolchain/v4/bin/virtualenv'
+    smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
+    cmake_generator: Ninja
+    make_command: ninja
```

Comment:
These settings are common between the 2 builders. Why not move them up into the builder template, and simply quote the expansion template here?

Reply:
There is one setting that is different -  `collection`.

Reply:
My bad, I think I get what you are saying.

Reply:
I think we can exclude the "collection" setting from the expansion template, and explicitly specify it right after quoting the template (like what's implemented in lines 83-85 below).

Reply:
I like that :). Updated in [f77d924](https://github.com/wiredtiger/wiredtiger/pull/8909/commits/f77d924a611b196ad162bb62eb9f463d52d53da2)

Reply:
Corrected commit updated in [f4f07b1](https://github.com/wiredtiger/wiredtiger/pull/8909/commits/f4f07b108ab02ff0b7ebb6c5697cbbf602537217)